Makefile

```diff
@@ -1,3 +1,21 @@
+.PHONY: \
+	clean \
+	functest \
+	govet \
```

Comment:
?

Reply:
To avoid a conflict with a file of the same name, and to improve performance. I found this convincing https://stackoverflow.com/questions/2145590/what-is-the-purpose-of-phony-in-a-makefile